build.xml

```diff
@@ -4,11 +4,19 @@
     <property name="path.root" value="${project.basedir}/project-base"/>
     <property name="path.vendor" value="${project.basedir}/vendor"/>
     <property name="path.packages" value="${project.basedir}/packages"/>
+    <property name="path.backend-api" value="${path.packages}/backend-api"/>
     <property name="path.framework" value="${path.packages}/framework"/>
     <property name="path.utils" value="${project.basedir}/utils"/>
     <property name="translations.dump.flags" value=""/>
 
     <import file="${path.framework}/build.xml"/>
+    <import file="${path.backend-api}/build.xml"/>
+
+    <target name="build-demo-ci" depends="backend-api-install,composer-dev,backend-api-oauth-keys-generate,shopsys_framework.build-demo-ci,backend-api-uninstall" description="Builds application for development with clean demo DB and runs CI checks."/>
+
+    <target name="build-demo-dev" depends="backend-api-install,composer-dev,backend-api-oauth-keys-generate,shopsys_framework.build-demo-dev,backend-api-uninstall" description="Builds application for development with clean demo DB and runs checks on changed files."/>
```

Comment:
I'm not happy with composer-dev running here and again in `shopsys_framework.build-demo-dev`.
But I understand why it's needed...

Reply:
before we can call `backend-api-oauth-keys-generate` we must be sure composer is installed, that's why.
Phing is also clever - it runs dependencies only once, so `composer-dev` runs only once - before `backend-api-oauth-key` and never again

Reply:
Oh really? That's nice!

Reply:
@grossmannmartin It's true, but only among dependencies of a single target. If you call multiple targets at once (via `phing target1 target2`) or use `<phingcall ... />` for a subcall, Phing will call the same target multiple times.